cupy/math/floating.py

```diff
@@ -50,6 +50,10 @@
     'nextafter', 2, 'cupy_nextafter',
     '''Computes the nearest neighbor float values towards the second argument.
 
+    .. note::
+        Result of :func:`cupy.nextafter` may be different from that of
+        :func:`numpy.nextafter` because CuPy sets ``-ftz=true``.
```

Comment:
Is it possible to add a bit more description here so that users can understand when/how much it is different from NumPy results? e.g. `for values are close to zero (or denormal numbers)`.